osfclient/cli.py

```diff
@@ -224,7 +224,10 @@ def list_(args):
             if path.startswith('/'):
                 path = path[1:]
 
-            print(os.path.join(prefix, path))
+            vals = [os.path.join(prefix, path)]
+            if args.long:
+                vals.append(str(file_.guid))
+            print(",".join(vals))
```

Comment:
There is the `csv` module which might be handy to take care of all the escaping of characters and stuff so that we don't become one of those tools that produce broken CSV.

Reply:
This one? https://pypi.python.org/pypi/csv

This seems a bit more fleshed out:
https://pypi.python.org/pypi/csvkit

any prefs? I only have experience of pandas which seems like a heavyweight dependency here.

Reply:
I was thinking of https://docs.python.org/3/library/csv.html